parser_test.go

```diff
@@ -1025,6 +1025,15 @@ func (s *testParserSuite) TestDBAStmt(c *C) {
 		{"set names utf8, @@session.sql_mode=1;", true, "SET NAMES 'utf8', @@SESSION.`sql_mode`=1"},
 		{"set @@session.sql_mode=1, names utf8, charset utf8;", true, "SET @@SESSION.`sql_mode`=1, NAMES 'utf8', NAMES 'utf8'"},
 
+		// for set/show config
+		{"set config TIKV LOG.LEVEL='info'", true, "SET CONFIG TIKV LOG.LEVEL = 'info'"},
+		{"set config PD LOG.LEVEL='info'", true, "SET CONFIG PD LOG.LEVEL = 'info'"},
+		{"set config TIDB LOG.LEVEL='info'", true, "SET CONFIG TIDB LOG.LEVEL = 'info'"},
+		{"set config '127.0.0.1:3306' LOG.LEVEL='info'", true, "SET CONFIG '127.0.0.1:3306' LOG.LEVEL = 'info'"},
```

Comment:
It'll be better to add some invalid test cases. Such as `set config tipd log.level = 'info'`, etc.

Reply:
It's designed to let the `TiDB` check if the `type` is valid instead of `Parser`, which is more extendable, for example, if we want to support changing `TiFlash` configs online, we don't need to modify the `Parser` again.